core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="admin-prod-mode-activities-tab">
+  <mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
+  The 'Activities' tab is not available in the production environment.
+  </mat-card>
+</div>
+<style>
+  .admin-prod-mode-activities-tab .oppia-long-text-card {
```

Comment:
I was expecting the earlier selector `admin-prod-mode-activities-tab .oppia-long-text-card` to work, is there any specific reason or this change?

Reply:
I found @srijanreddy98 requested for this change over [here](https://github.com/oppia/oppia/pull/11005#discussion_r508995250). I'll talk to him offline and update this thread accordingly.

Reply:
@DubeySandeep, should I change it to the earlier selector or keep this one?